.github/workflows/rust.yml

```diff
@@ -166,7 +166,7 @@ jobs:
         if: matrix.directory == 'packages/engine'
         uses: actions/setup-python@v2
         with:
-          python-version: "3.7"
+          python-version: "3.9"
```

Comment:
🎉 Very happy to see this.

Is there a reason we're not just upgrading to Python 3.10?